doc/pyproject_toml.rst

```diff
@@ -399,19 +399,23 @@ They are in the same format as the newer ``[project.entry-points]`` tables
 
 .. _pyproject_toml_sdist:
 
-Sdist section
--------------
+Contents of distribution files
+------------------------------
 
 .. versionadded:: 2.0
 
-With no configuration, Flit can make an sdist with everything it needs
-to build and install your module: the package contents (including non-Python
-data files, but not ``.pyc`` bytecode files), your ``pyproject.toml`` file,
-the readme & license files given in the metadata, and the :ref:`external data
-folder <pyproject_toml_external_data>` if you specified that.
+When building a wheel, Flit includes the package contents (including non-Python
+data files, but not ``.pyc`` bytecode files) along with the normal wheel
+metadata.
 
-If you want more control, you can give lists of paths or glob patterns as
-``include`` and ``exclude`` in this section. For example:
+When building an sdist, in addition to the package contents Flit includes
+everything it needs to build and install your module: your ``pyproject.toml``
+file, the readme & license files given in the metadata, and the :ref:`external
+data folder <pyproject_toml_external_data>` if you specified that.
+
+If you want more control over the content of the sdist, you can give lists of
```

Comment:
While the include & exclude lists are only used when building the sdist, the exclude list can affect wheels as well, because the common pattern (including in `build`) is to build the sdist first, unpack it, and build a wheel from that.

I'm not entirely sure of the best way to include this information, so feel free to leave it for me to address separately, but I thought it was worth mentioning while we're looking at this.

Reply:
I didn't explicitly point out that the *include* list can't affect the wheel (as it only adds non-package content), because I felt things could get too complicated if I did so. But again, let me know if you disagree.